Documentation/installing-dependencies.rst

```diff
@@ -1,216 +1,293 @@
 .. _spack_installation:
 
 Spack installation
 ------------------
 
-`Spack <https://github.com/spack/spack/wiki>`_ is a HPC software package manager.
-It automates the process of installing, upgrading, configuring, and removing computer programs.
-In particular, our spack package `seissol-env` allows automatically installing all dependencies of SeisSol (e.g. mpi, hdf5, netcdf, easi, asagi, etc).
+The `Spack <https://github.com/spack/spack/wiki>`_ package `seissol-env` allows to automatically install all dependencies of SeisSol (e.g. mpi, hdf5, netcdf, easi, asagi, etc),
+leaving only SeisSol itself left to be build.
+
 See https://github.com/SeisSol/seissol-spack-aid/tree/main/spack for details on the installation with spack.
 See also for reference our documentation on how to compile seissol-env on :ref:`SuperMUC-NG <compile_run_supermuc>`, :ref:`Shaheen <compile_run_shaheen>` (Cray system) and :ref:`Frontera <compile_run_frontera>`.
 
-
-Manual installation
+Manual Installation
 -------------------
 
-Initial Adjustments to .bashrc
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+We recommend setting up a folder on your system which will contain the build files for all dependencies.
+For example, do `mkdir ~/seissol; cd seissol`.
+
+In _all_ cases before building something manually,
+make sure to check the installed module files. That is, type `module avail` and look for the software you want to use.
+Type `module load NAME` to load the respective software (with `NAME` being the name of the software, including the text after the slash, e.g. `module load cmake/3.20.0`).
 
-Add the following lines to your .bashrc (vi ~/.bashrc).
+Setting Helpful Environment Variables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Create a file `setup.sh` with the following enviroment variables. The following script assumes that you use the folder `~/seissol`
+for building.
 
 .. code-block:: bash
 
   # For intel compiler
   # source /opt/intel/compiler/VERSION/bin/compilervars.sh intel64
   
-  export PATH=$HOME/bin:$PATH
-  export LIBRARY_PATH=$HOME/lib:$LIBRARY_PATH
-  export LD_LIBRARY_PATH=$HOME/lib:$LD_LIBRARY_PATH
-  export PKG_CONFIG_PATH=$HOME/lib/pkgconfig:$PKG_CONFIG_PATH
-  export CMAKE_PREFIX_PATH=$HOME
-  export EDITOR=vi
-  export CPATH=$HOME/include:$CPATH 
+  # write the path here which you created your directory in (you can figure it out by typing `pwd`)
+  export SEISSOL_BASE=$HOME/seissol
+
+  export PREFIX=$SEISSOL_BASE/local
+  export PATH=$PREFIX/bin:$PATH
+  export LIBRARY_PATH=$PREFIX/lib:$PREFIX/lib64:$LIBRARY_PATH
+  export LD_LIBRARY_PATH=$PREFIX/lib:$PREFIX/lib64:$LD_LIBRARY_PATH
+  export PKG_CONFIG_PATH=$PREFIX/lib/pkgconfig:$PREFIX/lib64/pkgconfig:$PKG_CONFIG_PATH
+  export CMAKE_PREFIX_PATH=$PREFIX:$CMAKE_PREFIX_PATH
+  export CMAKE_INSTALL_PREFIX=$PREFIX
+  export CPATH=$PREFIX/include:$CPATH 
+  export C_INCLUDE_PATH=$PREFIX/include:$C_INCLUDE_PATH
+  export CXX_INCLUDE_PATH=$PREFIX/include:$CXX_INCLUDE_PATH
+  export EDITOR=nano # or e.g. vi,vim
```

Comment:
nano?! But let's not start an editor war here ;-)

Reply:
It's the easiest-to-use. Otherwise, I agree—I'd prefer to write `vim` in there anyways... :D